components/validator/l2_output_submitter.go

```diff
@@ -165,19 +167,19 @@ func (l *L2OutputSubmitter) trySubmitL2Output() error {
 		return fmt.Errorf("failed to submit l2 output transaction: %w", err)
 	}
 	l.metr.RecordL2OutputSubmitted(output.BlockRef)
-	l.retryAfter(1 * time.Second)
+	l.retryAfter(l.cfg.OutputSubmitterRetryInterval)
```

Comment:
If a retryAfter call is needed even on a successful submit, it would seem to make sense to call retryAfter unconditionally in the trySubmitL2Output call.

Reply:
Thank you for your suggestion. I will mark it as TODO because the changes might reflect E2E tests currently fixing.
I will change `trySubmitL2Output` function to return wait duration explicitly, and handle `retryAfter` function calls at once using the return value.